rocAL/rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -371,7 +371,7 @@ rocalSequenceReaderSingleShard(
         bool loop,
         unsigned step,
         unsigned stride) {
-    rocALTensor* output = nullptr;
+    Tensor* output = nullptr;
     if (p_context == nullptr) {
```

Comment:
Why are we changing this from rocALTensor to Tensor?
The output of this api call is of type rocALTensor right?

Reply:
All internal references should be Tensor and not rocALTensor